server/src/main/java/org/apache/druid/metadata/SQLMetadataRuleManager.java

```diff
@@ -311,7 +311,8 @@ public void poll()
           )
       );
 
-      log.info("Polled and found rules for %,d datasource(s)", newRules.size());
+      int newRuleCount = newRules.values().stream().mapToInt(List::size).sum();
```

Comment:
nit: `newRuleCount` can be declared final.

Reply:
Of course. But it's just a local variable, why need to add final?